src/app/docs/navbar/navbar-docs.component.html

```diff
@@ -13,10 +13,57 @@
   </sky-docs-demo-page-summary>
 
   <sky-docs-demo>
-    See documentation for how to setup demos.
+    <sky-navbar>
+      <sky-navbar-item>
+        <a
+          routerLink="/"
+          routerLinkActive="sky-navbar-item-active"
+        >
+          Selected item
```

Comment:
When I click the `Selected item` item, it takes me to https://developer.blackbaud.com/skyux-navbar/. Is that the intended behavior?

Reply:
It was from the last code...but yeah that is a bit awkward. Lets change that to be the root of the docs page.

Reply:
Well, since these are router links there's really no where to go in the same SPA that would make sense. To avoid any confusion, I just linked it to the same page (/docs/navbar). I think the demo still holds up without the linking actually doing anything. Let me know if you have any ideas.

Reply:
That sound OK to me. When I checked the current demo, it doesn't seem like that option is clickable at all, so this seems like an improvement on that.